gremlin-test/src/main/resources/org/apache/tinkerpop/gremlin/test/features/filter/Range.feature

```diff
@@ -280,4 +280,16 @@ Feature: Step - range()
     When iterated to list
     Then the result should be unordered
       | result |
-      | d[29].i |
\ No newline at end of file
+      | d[29].i |
+
+  Scenario: g_V_mapXinX_limitX2X_valuesXnameX
+    Given the modern graph
+    And the traversal of
+      """
+      g.V().map(__.in()).limit(2).values('name')
```

Comment:
nit: we prefer double quotes for strings in our feature tests as a convention.

Reply:
done